doc-content/enterprise-only/openshift/environment-authoring-con.adoc

```diff
@@ -1,16 +1,43 @@
+Depending on your needs, you can deploy either a single authoring environment template or a high-availability (HA) authoring environment template.
+
+A single authoring environment contains two pods. One of the pods runs {CENTRAL}, the other runs {KIE_SERVER}. 
+ifdef::PAM[The {KIE_SERVER} includes an embedded H2 database engine.]
+This type of environment uses the least possible amount of resources. 
+
+An HA authoring environment contains several pods. Both {CENTRAL} and {KIE_SERVER} are provided in scalable pods that can run in parallel and share persistent storage. 
+ifdef::PAM[The database is provided by a separate pod.]
+Additional pods also contain the AMQ and Data Grid servers that enable high-availability functionality in {CENTRAL}. Use a high-availability authoring environment to provide maximum reliability and responsiveness, especially if several users are involved in authoring at the same time.
+
+In the current version of {PRODUCT}, an HA authoring environment is supported with certain limitations:
+
+* If one user is deploying a service to the {KIE_SERVER}, another user will not be able to deploy a service to the same {KIE_SERVER} until the first deployment completes.
 
-[IMPORTANT]
-====
-In {PRODUCT} {PRODUCT_VERSION}, high-availability {CENTRAL} functionality is for Technology Preview only. For more information on Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Scope].
-====
+* If a {CENTRAL} pod crashes while a user works with it, the user can get an error message and then is redirected to another pod. Logging on again is not required. 
+
+* If a {CENTRAL} pod crashes during a user operation, data that was not committed might be lost. 
+
+* If a {CENTRAL} pod crashes during creation of a project, an unusable project might be created. 
+
+* If a {CENTRAL} pod crashes during creation of an asset, the asset might be created but not indexed, so it can not be used. The user can open the asset in {CENTRAL} and save it again to make it indexed.
+
+ifdef::PAM[]
+You can also deploy additional managed or immutable {KIE_SERVERS}, if required. {CENTRAL} can automatically discover any {KIE_SERVERS} in the same namespace, including immutable {KIE_SERVERS} and managed {KIE_SERVERS}. This feature requires the `OpenShiftStartupStrategy` setting, which is enabled for all {KIE_SERVERS} except those deployed in a fixed managed infrastructure. For instructions about deploying managed {KIE_SERVERS} with the `OpenShiftStartupStrategy` setting enabled, see {URL_DEPLOYING_MANAGED_FREEFORM_ON_OPENSHIFT}[_{DEPLOYING_MANAGED_FREEFORM_ON_OPENSHIFT}_]. For instructions about deploying immutable {KIE_SERVERS}, see {URL_DEPLOYING_IMMUTABLE_ON_OPENSHIFT}[_{DEPLOYING_IMMUTABLE_ON_OPENSHIFT}_].
```

Comment:
This is also possible for DM

Reply:
The DM version mentions the possibility elsewhere in the text and, more importantly, this entire part about document links does not apply. I'll see if I can do some more PAM/DM unification here; before this version I had separate files in place for PAM and DM for this section.